module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -196,48 +196,48 @@ namespace module::input {
             [this](const std::list<std::shared_ptr<const RawSensors>>& sensors, const KinematicsModel& model) {
                 // If we need to reset the filter, do that here
                 if (reset_filter.load()) {
-                    Eigen::Vector3d acc   = Eigen::Vector3d::Zero();
-                    Eigen::Vector3d gyro  = Eigen::Vector3d::Zero();
-                    Eigen::Vector3d rMFt  = Eigen::Vector3d::Zero();
-                    auto filtered_sensors = std::make_unique<Sensors>();
+                    Eigen::Vector3d acc  = Eigen::Vector3d::Zero();
+                    Eigen::Vector3d gyro = Eigen::Vector3d::Zero();
+                    Eigen::Vector3d rMFt = Eigen::Vector3d::Zero();
 
                     for (const auto& s : sensors) {
+                        auto filtered_sensors = Sensors();
```

Comment:
```suggestion
                        Sensors filtered_sensors;
```

Reply:
I mean, at least call the ctor explicitly :p 
```suggestion
                        Sensors filtered_sensors{};
```

Reply:
difference being?